ansible/config_sonic_basedon_testbed.yml

```diff
@@ -144,6 +144,41 @@
         become: true
         when: stat_result.stat.exists is defined and stat_result.stat.exists
 
+      - name: read account key
+        set_fact:
+            core_key: "{{ corefile_uploader['azure_sonic_core_storage']['account_key'] | default('') }}"
```

Comment:
what if corefile_uploader is not defined? will this fail the job?

Reply:
Yep it fails the job. It fails for any missing field in the path, except for last key, which is saved by default. Thanks for catching. I only tested for incorrect value for last key in the path.